src/cleed++/Core/Element.hh

```
@@ -0,0 +1,107 @@
+    static const std::map<int, std::tuple<std::string, std::string>> GROUPS;
+    static const std::map<int, std::string> DESCRIPTIONS;
+
+    Element(int Z, const std::string &symbol, const std::string &name);
+    virtual ~Element();
+
+    int getGroup() const;
+    int getPeriod() const;
+    char getBlock() const;
+    int getSeries() const;
+    double getMolarMass() const;
+    double getElectronegivity() const;
```

Clang (14.0.6)

[warning] function 'getElectronegivity' should be marked [[nodiscard]]

(modernize-use-nodiscard)